Improvements to Ultimate profile

TakerRed opened the pull request:
Changes:
- Buffed walk speed.
- Ftilt out of run now works better with the buffed walk speed.
- Removed isB modifier on ModX.
- Pressing Down + ModY no longer makes you fall through platforms when on it. This makes dtilts on platforms a bit easier.
- ModY no longer makes you fastfall/tap jump.
- Walking with ModY no longer transitions to run when releasing ModY. This makes it easier to control your character.

Changes that I think should be implemented, but haven't due to mixed opinions:
- Nerfs to turn around neutral B, and pikachu/pichu double up zip when pressing ModY. The isB modifier on ModY should be deleted in my opinion.

Commit:
Improvements to Ultimate profile

- Buffed walk speed.
- Ftilt out of run now works better with the buffed walk speed.
- Removed isB modifier on ModX.
- Pressing down + ModY no longer makes you fall through platforms when on it. This makes dtilts on platforms a bit easier.
- ModY no longer makes you fastfall/tap jump.
- Walking with ModY no longer transitions to run when releasing ModY. This makes it easier to control your character.

Commit:
Improvements to Ultimate profile

- Buffed walk speed.
- Ftilt out of run now works better with the buffed walk speed.
- Removed isB modifier on ModX.
- Pressing down + ModY no longer makes you fall through platforms when on it. This makes dtilts on platforms a bit easier.
- ModY no longer makes you fastfall/tap jump.
- Walking with ModY no longer transitions to run when releasing ModY. This makes it easier to control your character.

Kelvinrr commented on code/GCCPCB2-v1.203/GCCPCB2-v1.203.ino:
```diff
@@ -367,21 +370,20 @@ void loop()
         if (currentGame == PM) controlY = 128 + (positionY * 34);
       }
-
+      if (isA)
+        if (currentGame == Ultimate) controlX = 128 + (positionX * 36);
+      }
```
Missing an open curly bracket

Kelvinrr reviewed (commented):
Small syntax error preventing this from compiling.

Commit:
Update GCCPCB2-v1.203.ino

Fixed syntax error

Commit:
Added Brook Settings

- Hold Z while plugging in with the Brook PCB
- C-Up --> Up | C-Left --> Left | C-Right --> Right | C-Down --> Down
- Issue: Alternate movement buttons won't register until held down long enough. Inputs will also be delayed.
